utilities/src/main/java/io/syndesis/qe/utils/DropBoxUtils.java

```diff
@@ -0,0 +1,92 @@
+package io.syndesis.qe.utils;
+
+import com.dropbox.core.DbxException;
+import com.dropbox.core.DbxRequestConfig;
+import com.dropbox.core.v2.DbxClientV2;
+import com.dropbox.core.v2.files.FileMetadata;
+import com.dropbox.core.v2.files.GetMetadataErrorException;
+import io.syndesis.qe.accounts.Account;
+import io.syndesis.qe.accounts.AccountsDirectory;
+import javassist.tools.rmi.ObjectNotFoundException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+
+@Slf4j
+@Component
+public class DropBoxUtils {
+    private final DbxClientV2 client;
+
+    public DropBoxUtils() throws ObjectNotFoundException, DbxException {
+        Optional<Account> optional = AccountsDirectory.getInstance().getAccount("QE Dropbox");
+
+        if (optional.isPresent()) {
+            DbxRequestConfig config = new DbxRequestConfig(optional.get().getProperty("clientIdentifier"));
+            this.client = new DbxClientV2(config, optional.get().getProperty("accessToken"));
+        } else {
+            throw new ObjectNotFoundException("DropBox credentials were not found.");
```

Comment:
Log error with explanation and maybe IllegalArg is more suitable here as `ObjectNotFoundException` is not Runtime ex.